ansible_collections/arista/avd/roles/eos_validate_state/tasks/ping_inband.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Gather fabric reachability from Management Interface {{ management_interface.key }}
+  eos_command:
+    commands: "ping {{ loopback0_address | ipaddr('address') }} repeat 1 interface {{ management_interface.key }}"
+  loop: "{{ loopback0_reachability.loopback0_range }}"
+  loop_control:
+    loop_var: loopback0_address
+  ignore_errors: "{{ eos_validate_state_validation_mode_loose }}"
+  when: |
+    (loopback0_reachability.loopback0_range is defined and loopback0_reachability.loopback0_range is not none) and
+    (management_interface.value.type == "inband" )
```

Comment:
the type variable may not be set, so we should use `management_interface.value.type is arista.avd.defined("inband")`. Please also change the check for `loopback0_reachability.loopback0_range is arista.avd.defined`

Reply:
Solved